access-common/src/main/java/edu/unc/lib/dl/ui/service/SolrQueryLayerService.java

```diff
@@ -710,7 +712,7 @@ public HierarchicalBrowseResultResponse getHierarchicalBrowseResults(Hierarchica
 					this.generateSearch(filteredChildrenRequest));
 
 			this.getChildrenCounts(results.getResultList(), accessGroups, "containers",
-					"contentModel:" + SolrSettings.sanitize(ContentModelHelper.Model.CONTAINER.toString()),
+					"resourceType:" + ("(" + ResourceType.Collection.toString() + " OR " + ResourceType.Folder.toString() + ")"),
```

Comment:
I find the ('s wrapping the "("'s kind of confusing.  Its totally cosmetic and up to you, but removing the outter ('s shouldn't cause any problems.